apps/voting/contracts/Voting.sol

```diff
@@ -489,6 +496,25 @@ contract Voting is IForwarder, AragonApp {
         }
     }
 
+    /**
+     * @notice Return the voting power of the `_voters` at the current block
+     * @param _voters list of voters
+     * @return the array of governance token balances
+     */
+    function getVotingPowerMultiple(address[] _voters) external view returns (uint256[] memory balances) {
+        return _getVotingPowerMultipleAt(_voters, getBlockNumber64());
+    }
+
+    /**
+     * @notice Return the voting power of the `_voters` at the vote #`_voteId` snapshot block
+     * @param _voteId Vote identifier
+     * @param _voters list of voters
```

Comment:
Start with a capitalized letter after the variable name, consistent with the rest of the contract.

Reply:
Addressed in 86fd9f855f8a47a06e01630a240519fdc71f4afa.